### PR TITLE
CNAPP-5525-IsraelRegion

### PR DESCRIPTION
**Pull request by kerenshm**

### CODE REVIEW

File: dome9/common/providerconst/const.go [hunk 1]
```diff
@@ -124,7 +125,7 @@ var AzureSecurityGroupProtocol = []string{"UDP", "TCP", "ANY"}
 var AzureSecurityGroupSourceScopeTypes = []string{"CIDR", "IPList", "Tag"}
 
 // The 21 regions Dome9 manages in AWS cloud account
-var AWSRegions = []string{"us_east_1", "us_west_1", "eu_west_1", "ap_southeast_1", "ap_northeast_1", "us_west_2", "sa_east_1", "ap_southeast_2", "eu_central_1", "ap_northeast_2", "ap_south_1", "us_east_2", "ca_central_1", "eu_west_2", "eu_west_3", "eu_north_1", "ap_east_1", "me_south_1", "af_south_1", "eu_south_1", "ap_northeast_3", "me_central_1", "ap_south_2", "ap_southeast_3", "ap_southeast_4", "eu_central_2", "eu_south_2"}
+var AWSRegions = []string{"us_east_1", "us_west_1", "eu_west_1", "ap_southeast_1", "ap_northeast_1", "us_west_2", "sa_east_1", "ap_southeast_2", "eu_central_1", "ap_northeast_2", "ap_south_1", "us_east_2", "ca_central_1", "eu_west_2", "eu_west_3", "eu_north_1", "ap_east_1", "me_south_1", "af_south_1", "eu_south_1", "ap_northeast_3", "me_central_1", "ap_south_2", "ap_southeast_3", "ap_southeast_4", "eu_central_2", "eu_south_2", "il-central-1"}
```
Comment (chkp-dolevh) [inline dome9/common/providerconst/const.go]: should be:
il_central_1

File: dome9/common/providerconst/const.go [hunk 2]
```diff
@@ -115,6 +115,7 @@ const (
 	AP_SOUTHEAST_4	    = "32"
 	EU_CENTRAL_2	    = "33"
 	EU_SOUTH_2	    = "34"
+	IL_CENTRAL_1 =  "35"
```
Comment (chkp-dolevh) [inline dome9/common/providerconst/const.go]: please add spaces to align with other region's

**Review by chkp-dolevh — CHANGES_REQUESTED** [review]
Please check in depth if you don't need to add this region in other places as well

File: dome9/common/providerconst/const.go (hunk 1 above)
Comment (kerenshm) [reply dome9/common/providerconst/const.go]: fixed

File: dome9/common/providerconst/const.go (hunk 2 above)
Comment (kerenshm) [reply dome9/common/providerconst/const.go]: fixed

File: website/docs/d/role.html.markdown [hunk 3]
```diff
@@ -45,6 +45,6 @@ In addition to all arguments above, the following attributes are exported:
 ### SRL 
 * `type` - (Optional) Accepted values: AWS, Azure, GCP, OrganizationalUnit.
 * `main_id` - (Optional) Cloud Account or Organizational Unit ID.
-* `region` - (Optional) Accepted values: "us_east_1", "us_west_1", "eu_west_1", "ap_southeast_1", "ap_northeast_1", "us_west_2", "sa_east_1", "ap_southeast_2", "eu_central_1", "ap_northeast_2", "ap_south_1", "us_east_2", "ca_central_1", "eu_west_2", "eu_west_3", "eu_north_1", "ap_east_1", "me_south_1", "af_south_1", "eu_south_1", "ap_northeast_3", "me_central_1", "ap_south_2", "ap_southeast_3", "ap_southeast_4", "eu_central_2", "eu_south_2".
+* `region` - (Optional) Accepted values: "us_east_1", "us_west_1", "eu_west_1", "ap_southeast_1", "ap_northeast_1", "us_west_2", "sa_east_1", "ap_southeast_2", "eu_central_1", "ap_northeast_2", "ap_south_1", "us_east_2", "ca_central_1", "eu_west_2", "eu_west_3", "eu_north_1", "ap_east_1", "me_south_1", "af_south_1", "eu_south_1", "ap_northeast_3", "me_central_1", "ap_south_2", "ap_southeast_3", "ap_southeast_4", "eu_central_2", "eu_south_2", "il_central_1".
```
Comment (chkp-dolevh) [inline website/docs/d/role.html.markdown:48]: Please add it also to:
website/docs/r/role.html.markdown

**Review by chkp-dolevh — APPROVED** [review]

**Pull request merged**